requirements.txt

```diff
@@ -15,7 +15,7 @@ matplotlib
 holidays
 tornado
 # security plugin
-pycrypto
+pycryptodome
```

Comment:
Is this required to make the mac build work?

Reply:
Negative, I can remove. Was needed for windows